tests/test_states/test_isotropic.py

```diff
@@ -11,7 +11,3 @@ def test_isotropic_qutrit():
     np.testing.assert_equal(np.isclose(res[0, 0], 2 / 9), True)
     np.testing.assert_equal(np.isclose(res[4, 4], 2 / 9), True)
     np.testing.assert_equal(np.isclose(res[8, 8], 2 / 9), True)
```

Comment:
I think these lines could also use `assert_array_equal` instead of comparing a few selected coefficients. It's up to you if you want to accept/reject the suggestion.

Reply:
Yep, good point! Ended up putting the whole matrix in as a check.